plugins/woocommerce-admin/webpack.config.js

```diff
@@ -44,49 +60,16 @@ const wcAdminPackages = [
 	'product-editor',
 	'remote-logging',
 ];
-// wpAdminScripts are loaded on wp-admin pages outside the context of WooCommerce Admin
-// See ./client/wp-admin-scripts/README.md for more details
-const wpAdminScripts = [
-	'marketing-coupons',
-	'onboarding-homepage-notice',
-	'onboarding-product-notice',
-	'onboarding-product-import-notice',
-	'onboarding-tax-notice',
-	'print-shipping-label-banner',
-	'beta-features-tracking-modal',
-	'payment-method-promotions',
-	'onboarding-load-sample-products-notice',
-	'product-tracking',
-	'add-term-tracking',
-	'attributes-tracking',
-	'category-tracking',
-	'tags-tracking',
-	'product-tour',
-	'wc-addons-tour',
-	'settings-tracking',
-	'order-tracking',
-	'product-import-tracking',
-	'variable-product-tour',
-	'product-category-metabox',
-	'shipping-settings-region-picker',
-	'command-palette',
-	'command-palette-analytics',
-	'woo-connect-notice',
-	'woo-plugin-update-connect-notice',
-	'woo-enable-autorenew',
-	'woo-renew-subscription',
-	'woo-subscriptions-notice',
-	'woo-product-usage-notice',
-];
+
```

Comment:
Without the explicit inclusion anymore, it would be good to update the [readme](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/wp-admin-scripts/README.md). We don't need to add them manually anymore so maybe a note on how its built would be helpful because we are always adding new ones

Reply:
Thanks for the reminder! I've updated the readme